src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/EmitHelpers.vb

```diff
@@ -51,7 +51,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             Catch e As NotSupportedException
                 ' TODO: https://github.com/dotnet/roslyn/issues/9004
                 diagnostics.Add(ERRID.ERR_ModuleEmitFailure, NoLocation.Singleton, compilation.AssemblyName, e.Message)
-                Return New EmitDifferenceResult(success:=False, diagnostics:=diagnostics.ToReadOnlyAndFree(), baseline:=Nothing, updatedMethods:=updatedMethods.ToImmutableAndFree(), updatedTypes:=updatedTypes.ToImmutableAndFree())
+                Return New EmitDifferenceResult(success:=False, diagnostics:=diagnostics.ToReadOnlyAndFree(), baseline:=Nothing, updatedMethods:=updatedMethods.ToImmutableAndFree(), changedTypes:=updatedTypes.ToImmutableAndFree())
```

Comment:
Same comment as in corresponding C# code.
Also, should `updatedTypes` be renamed to `changedTypes` here?